framework/model/module/module.go

```diff
@@ -49,6 +49,11 @@ type configH struct {
 // ModuleOptions carries the framework context for setting a module.
 // For fields marked with REQUIRED, the framework will ensure their existence,
 // and the module can be used directly.
+// NOTE: the Manager and the LeaderElectionCbs share the election status.
+// When switching from leader to candidate, the Manager will exit.
+// The framework will close the LeaderElection and ends the process after
+// the Manager exits. Therefore, when implementing a module, the scenario of
+// becoming the leader again may not be considered.
```

Comment:
Considering the relatively small number of state transition scenarios we actually support, I thought it would be better to just state it explicitly.

For example.

1. START -> standby -> master

2. START -> master -> {abnormally quit, renew failure -> quit}

Instead of: `-> master -> standby -> master -> ...` like cases